recipes/zmqpp/all/patches/0001-fix_zmqpp_export_header.patch

```diff
@@ -0,0 +1,24 @@
+diff -u a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
This way we know it's been fixed in the main branch

```suggestion
https://github.com/zeromq/zmqpp/commit/05ad2f3255e5d74febbd1e17663bbfc2ded46c18
diff -u a/CMakeLists.txt b/CMakeLists.txt
```

Reply:
Ok